.prettierrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "singleQuote": true,
+  "semi": false,
+  "trailingComma": "none"
```

Comment:
trailingCommas are a good thing - allows for easy cut and paste re-ordering - this is why they have been added to JS